src/claim.rs

```diff
@@ -26,6 +26,10 @@ pub struct Claim {
     pub result: Response,
     /// A ulid to track the revision
     pub revision: String,
+    /// A canonical reference to the bundle used in the last action.
```

Comment:
```suggestion
    /// An optional canonical reference to the bundle used in the last action.
```

Reply:
If this field is optional, could we represent it as `Option<String>` instead?

Reply:
It either already is or it has already been updated to be so.

Reply:
Oh, you're right.